index.bs

```diff
@@ -313,28 +314,25 @@ Datagrams are encrypted and congestion controlled.
 <pre class="idl">
 interface mixin DatagramTransport {
     readonly attribute unsigned short maxDatagramSize;
-    WritableStream sendDatagrams();
-    ReadableStream receiveDatagrams();
+    /* both streams are streams of Uint8Array objects */
+    readonly attribute ReadableStream readableDatagrams;
```

Comment:
Sorry to chime in so late, but I wonder if `datagramReadable` and `datagramWritable` would be clearer?

Reply:
It would.  Renamed them.